api/src/services/uploads/uploads.test.ts

```diff
@@ -203,3 +211,45 @@ describe('downloads', () => {
     )
   })
 })
+
+describe('getUploads', () => {
+  scenario('returns most recent upload for one category',
+    async (scenario: UploadScenario) => {
+      mockCurrentUser(scenario.user.one)
+      await updateOrganization({
```

Comment:
Could we pull out these `await` calls here and in the scenario below into a `beforeEach` block?